projects/ccd-case-ui-toolkit/src/lib/shared/components/palette/linked-cases/read-linked-cases-field.component.spec.ts

```diff
@@ -0,0 +1,551 @@
+              show_condition: ''
+            }
+          ]
+        }
+      }
+    }
+  };
+
+  const mockRouter = {
+    navigate: jasmine.createSpy('navigate'),
+    url: ''
+  };
+
+  const linkedCases: CaseLink[] = [
+    {
+      caseReference: '1682374819203471',
+      reasons: [],
+      createdDateTime: '',
+      caseType: 'SSCS',
+      caseTypeDescription: 'SSCS case type',
+      caseState: 'state',
+      caseStateDescription: 'state description',
+      caseService: 'Tribunal',
+      caseName: 'SSCS 2.1'
+    },
+    {
+      caseReference: '1682897456391875',
+      reasons: [],
+      createdDateTime: '',
+      caseType: 'SSCS',
+      caseTypeDescription: 'SSCS case type',
+      caseState: 'state',
+      caseStateDescription: 'state description',
+      caseService: 'Tribunal',
+      caseName: 'SSCS 2.1'
+    }
+  ];
+
+  const linkCaseReasons: LovRefDataByServiceModel = {
+    list_of_values: [
+      {
+        key: 'progressed',
+        value_en: 'Progressed as part of this lead case',
+        value_cy: '',
+        hint_text_en: 'Progressed as part of this lead case',
+        hint_text_cy: '',
+        lov_order: 1,
+        parent_key: null,
+        category_key: 'caseLinkReason',
+        parent_category: '',
+        active_flag: 'Y',
+        child_nodes: null,
+        from: 'exui-default',
+      },
+      {
+        key: 'bail',
+        value_en: 'Bail',
+        value_cy: '',
+        hint_text_en: 'Bail',
+        hint_text_cy: '',
+        lov_order: 2,
+        parent_key: null,
+        category_key: 'caseLinkReason',
+        parent_category: '',
+        active_flag: 'Y',
+        child_nodes: null,
+        from: 'exui-default',
+      },
+      {
+        key: 'other',
+        value_en: 'Other',
+        value_cy: '',
+        hint_text_en: 'Other',
+        hint_text_cy: '',
+        lov_order: 3,
+        parent_key: null,
+        category_key: 'caseLinkReason',
+        parent_category: '',
+        active_flag: 'Y',
+        child_nodes: null,
+        from: 'exui-default',
+      },
+    ]
+  };
+
+  linkedCasesService = {
+    caseId: '1682374819203471',
+    linkedCases,
+    getAllLinkedCaseInformation() {}
+  };
+
+  beforeEach(waitForAsync(() => {
+    appConfig = createSpyObj<AbstractAppConfig>('appConfig', ['getRDCommonDataApiUrl']);
+    commonDataService = createSpyObj('commonDataService', ['getRefData']);
+    TestBed.configureTestingModule({
+      imports: [
+        RouterTestingModule
+      ],
+      schemas: [CUSTOM_ELEMENTS_SCHEMA],
+      providers: [
+        { provide: ActivatedRoute, useValue: mockRoute },
+        { provide: Router, useValue: mockRouter },
+        { provide: CommonDataService, useValue: commonDataService },
+        { provide: AbstractAppConfig, useValue: appConfig },
+        { provide: LinkedCasesService, useValue: linkedCasesService },
+      ],
+      declarations: [ReadLinkedCasesFieldComponent],
+    })
+      .compileComponents();
```

Comment:
@johnbenjamin-hmcts - check indentation

Reply:
@Daniel-Lam I have updated the code now.